src/text.md

```diff
@@ -105,13 +105,15 @@ Some password managers will alert you if you have the same password for multiple
 
 Each factor in two-factor authentication is a different way to prove you're you. Typically, factors fall into three types: something you *know* (like a password or the answer to a security question), something you *have* (like a phone or a key), and something you *are* (like a fingerprint or a retina scan). By using more than one factor, it makes it harder for someone else to pretend to be you.
 
 If you have an ATM card, you're already using 2FA: the card is something you *have*, and your PIN is something you *know*. If your house has an alarm, that's 2FA too: you *have* your key, and you *know* your alarm code. In fact, something you *know* plus something you *have* is the most common form of 2FA. For online accounts, the thing you *know* will be your password, and the thing you *have* will be your phone, where, after logging in with your username and password, you'll be asked to enter a code either from an authenticator app or from a text message that the service sends you.
 
-The system used by authenticator apps for 2FA is a standard, so any of the major authenticator apps will be compatible with it: [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&gl=US), [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator), [Authy](https://authy.com/), and [Aegis](https://getaegis.app/) (which gets a bonus point for being open source) will all work exactly the same way. Pick one and use it for all of your 2FA-enabled services. 1Password and LastPass build a 2FA app right into their password managers, so if you're already using one of them, you don't need an extra app. Because the system for 2FA apps is a standard, even if a service specifically says it uses (for example) Google Authenticator, you can use any one of these apps and it'll work just fine.
+The system used by authenticator apps for 2FA is a standard, so any of the major authenticator apps will be compatible with it: Google Authenticator [Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&gl=US) [iOS](https://apps.apple.com/us/app/google-authenticator/id388497605), [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator), [Authy](https://authy.com/), and [Aegis](https://getaegis.app/) (which gets a bonus point for being open source) will all work exactly the same way. Pick one and use it for all of your 2FA-enabled services. 1Password and LastPass build a 2FA app right into their password managers, so if you're already using one of them, you don't need an extra app. Because the system for 2FA apps is a standard, even if a service specifically says it uses (for example) Google Authenticator, you can use any one of these apps and it'll work just fine.
```

Comment:
I think it would look better for these to be in parentheses:

> Google Authenticator (Android, iOS), Microsoft Authenticator, Authy

as opposed to

> > Google Authenticator Android iOS, Microsoft Authenticator, Authy

Reply:
done